core/src/types/metadata.rs

```diff
@@ -196,17 +231,17 @@ impl Metadata {
     /// Set Content Type of this entry.
     ///
     /// Content Type is defined by [RFC 9110](https://httpwg.org/specs/rfc9110.html#field.content-type).
-    pub fn set_content_type(&mut self, v: &str) -> &mut Self {
-        self.content_type = Some(v.to_string());
+    pub fn set_content_type(&mut self, content_type: &str) -> &mut Self {
```

Comment:
`v` is clear enough. I don't think we need to change into it's full name.

Reply:
Just want to keep it consistent with the other items. 👀

Reply:
> Just want to keep it consistent with the other items.

Oh, please change all other items into the same `v` style.